Ginger/GingerCore/Actions/ActDBValidation.cs

```diff
@@ -133,11 +133,13 @@ public enum eDatabaseTye
         public eDatabaseTye DatabaseType
         {
             get
-            {try
+            {
+                try
                 {
-                    if (DB.DBType == Database.eDBTypes.Cassandra || DB.DBType == Database.eDBTypes.Couchbase || DB.DBType == Database.eDBTypes.MongoDb)
+                    if (DB != null && (DB.DBType == Database.eDBTypes.Cassandra || DB.DBType == Database.eDBTypes.Couchbase || DB.DBType == Database.eDBTypes.MongoDb))
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5848147)